### ISSUE TITLE
iOS15.2, Watch 8.3 - Shortcuts keeps asking for permission on Watch

### ISSUE THREAD

**Issue by yazz007** [home-assistant/iOS#2037]
Model Name: Watch 6, iPhone 11
Software Version: 8.3, iOS 15.2
App version: 2021.12.1

**Home Assistant Core Version**
2021.11.1

**Describe the bug**
Shortcuts keeps asking for permission every time when HomeAssitant is installed on Watch. This doesn't happen when HomeAssistant is not on Watch, then it only asks for the first time and no more. I tried several times and this pattern is stable.

**To Reproduce**
See above

**Expected behavior**
Permission notification for Shortcut running HomeAssistant should pop up only once.

**Comment by ProfDrYoMan** [comment]
This started with 15.0 or the corresponding watchOS (8.x?) update.

If I uninstall HA watch app I can use Siri and Shortcuts again?

**Comment by krissen** [comment]
Duplicate of #1925 in the sense that what is described initially in #1925 is how the issue first presented itself. In later versions of iOS/watchOS the app doesn't report "null" anymore, but the end result is the same: asking permissions every time.
